spot_wrapper/spot_arm.py

```diff
@@ -483,11 +484,14 @@ def hand_pose(
             # Rotation as a quaternion.
             rotation = geometry_pb2.Quaternion(w=qw, x=qx, y=qy, z=qz)
 
-            traj_duration = seconds_to_duration(duration)
-
             # Build the SE(3) pose of the desired hand position in the moving body frame.
             hand_pose = geometry_pb2.SE3Pose(position=position, rotation=rotation)
-            hand_pose_traj_point = trajectory_pb2.SE3TrajectoryPoint(pose=hand_pose, time_since_reference=traj_duration)
+            hand_pose_traj_point = trajectory_pb2.SE3TrajectoryPoint(pose=hand_pose)
+            if duration != 0.0:
```

Comment:
Generally skeptical of floating point equality checks. If it were up to me, I'd maybe make `duration` be a `double | None`, so you can specifically set it to `None` or something.

Reply:
Updated to be optionally None. Thanks for catching that, I wasn't really going through everything with a fine toothcomb